README.MacOS.md

```diff
@@ -10,7 +10,20 @@ Download and install the following tools:
 *  [Homebrew](http://mxcl.github.com/homebrew/).
 *   Any emacs distribution, for instance [Aquamacs](http://aquamacs.org/).
 
-### Homebrew tap
+### Homebrew cask
+
+In case you don't have plans to work on Mozart 2, you can simply install it as
+a compiled binary via [Homebrew cask] (if you have Homebrew installed you're
+already set up):
```

Comment:
This text disappears in the rendered version. Maybe just move it to the next line.